service/src/main/java/bio/terra/profile/db/ProfileChangeLogDao.java

```diff
@@ -0,0 +1,154 @@
+package bio.terra.profile.db;
+
+import bio.terra.common.db.ReadTransaction;
+import bio.terra.common.db.WriteTransaction;
+import bio.terra.common.exception.SerializationException;
+import bio.terra.profile.model.ChangeLogEntry;
```

Comment:
Similar to my other comment: This appears to be an API model, an internal representation should be used at the DAO layer and then marshaled to an API representation further up the stack.